solc/CommandLineParser.cpp

```diff
@@ -637,7 +637,8 @@ General Information)").c_str(),
 			po::value<std::string>()->default_value(util::toString(DebugInfoSelection::Default())),
 			("Debug info components to be included in the produced EVM assembly and Yul code. "
 			"Value can be all, none or a comma-separated list containing one or more of the "
-			"following components: " + util::joinHumanReadable(DebugInfoSelection::componentMap() | ranges::views::keys) + ".").c_str()
+			"following components: " + util::joinHumanReadable(DebugInfoSelection::componentMap() | ranges::views::keys) + ". "
+			"Note that ethdebug is highly experimental - everything may change.").c_str()
```

Comment:
Another option would be to just filter out ``ethdebug`` here and leave it undocumented until we stabilize it more.

Reply:
True! Thats much better!